%YAML:1.0
---

nodes:
  #Is VICON Node
  0:
    has_uwb: true
    has_armarkers: false
    has_camera: false
    has_vo: false
    has_global_pose: false
    is_static: true
    anntena_pos: [0, 0, 0]
  1:
    has_uwb: true
    has_armarkers: true
    has_camera: true
    has_vo: true
    is_static: false
    has_global_pose: false
    anntena_pos: [0, -0.083, 0.078]

  6:
    has_uwb: true
    has_armarkers: true
    has_camera: true
    has_vo: true
    has_global_pose: false
    is_static: false
    anntena_pos: [0, -0.083, 0.078]

  #For PC Command line
  7:
    has_uwb: true
    has_armarkers: true
    has_camera: true
    has_vo: true
    has_global_pose: false
    is_static: true
    anntena_pos: [0.0, 0.0, 0.0]
